sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -337,39 +335,45 @@ describe("Test ServiceBusClient creation", function(): void {
     should.equal(errorWasThrown, true, "Error thrown flag must be true");
   });
 
-  // if (isNode) {
-  //   it("Coerces input to string for host in credential based constructor", async function(): Promise<
-  //     void
-  //   > {
-  //     const tokenCreds = getDefaultTokenCredential();
-  //     sbClient = new ServiceBusClient(123 as any, tokenCreds);
-  //     should.equal(sbClient.name, "sb://123/", "Name of the namespace is different than expected");
-  //   });
-
-  //   it("sends a message to the ServiceBus entity", async function(): Promise<void> {
-  //     const tokenCreds = getDefaultTokenCredential();
-
-  //     const serviceBusClient = createServiceBusClientForTests();
-  //     const entities = await serviceBusClient.test.createTestEntities(
-  //       TestClientType.UnpartitionedQueue
-  //     );
-  //     await serviceBusClient.close();
-
-  //     const sbClient = new ServiceBusClient(serviceBusEndpoint, tokenCreds);
-  //     sbClient.should.be.an.instanceof(ServiceBusClient);
-
-  //     const sender = sbClient.createSender(entities.queue!);
-  //     const receiver = await sbClient.createReceiver(entities.queue!, "peekLock");
-  //     const testMessages = TestMessage.getSample();
-  //     await sender.send(testMessages);
-  //     const msgs = await receiver.receiveBatch(1);
-
-  //     should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
-  //     should.equal(msgs[0].body, testMessages.body, "MessageBody is different than expected");
-  //     should.equal(msgs.length, 1, "Unexpected number of messages");
-  //     await sbClient.close();
-  //   });
-  // }
+  if (isNode) {
+    it("throws error for invalid host name", async function(): Promise<void> {
+      try {
+        new ServiceBusClient(123 as any, getDefaultTokenCredential());
+      } catch (error) {
+        errorWasThrown = true;
+        should.equal(
+          error.message,
+          "`host` parameter is not a string",
+          "ErrorMessage is different than expected"
+        );
+      }
+      should.equal(errorWasThrown, true, "Error thrown flag must be true");
+    });
+
+    it("sends a message to the ServiceBus entity", async function(): Promise<void> {
+      const tokenCreds = getDefaultTokenCredential();
+
+      const serviceBusClient = createServiceBusClientForTests();
+      const entities = await serviceBusClient.test.createTestEntities(
+        TestClientType.UnpartitionedQueue
+      );
+      await serviceBusClient.close();
+
+      const sbClient = new ServiceBusClient(serviceBusEndpoint, tokenCreds);
+      sbClient.should.be.an.instanceof(ServiceBusClient);
```

Comment:
Why are we testing this?

Reply:
Lol, I don't know. It was there from the beginning, I can remove.